hnn_core/viz.py

```diff
@@ -523,10 +523,15 @@ def plot_spikes_raster(cell_response, trial_idx=None, ax=None, show=True):
         spike_times = np.array([])
         spike_types = np.array([])
         spike_gids = np.array([])
+    cell_types = cell_response._cell_type_names
+    cell_type_colors = ['r', 'b', 'g', 'w']
+    # Ensure cell_types list matches the size of cell_type_colors dictionary
+    if len(cell_types) != len(cell_type_colors):
+        raise ValueError("cell_types list must "
+                         "match the number of items in cell_type_colors.")
 
-    cell_types = ['L2_basket', 'L2_pyramidal', 'L5_basket', 'L5_pyramidal']
-    cell_type_colors = {'L5_pyramidal': 'r', 'L5_basket': 'b',
-                        'L2_pyramidal': 'g', 'L2_basket': 'w'}
+    # Create a new dictionary with the new keys and old values
+    new_cell_type_colors = dict(zip(cell_types, cell_type_colors))
```

Comment:
Maybe we should use a default colormap here and ask users to enter a dict of colors if they want a specific cell_type -> color mapping. @ntolley wdyt?